pkg/controller.v1/tensorflow/job.go

```diff
@@ -225,21 +225,28 @@ func (tc *TFController) cleanupTFJob(tfJob *tfv1.TFJob) error {
 		return nil
 	}
 	duration := time.Second * time.Duration(*ttl)
-	if currentTime.After(tfJob.Status.CompletionTime.Add(duration)) {
+	if tfJob.Status.CompletionTime == nil {
```

Comment:
Why is this change necessary?

Reply:
Optimized the performance of the operator to avoid unnecessary reconciles of completed tfjob.